src/Views/Auth.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import AuthForm from '../components/AuthForm';
+import { signInUser, signUpUser } from '../services/users';
+import classNames from 'classnames';
+import './Auth.css';
+
+export default function Auth({ setCurrentUser }) {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [type, setType] = useState('signin');
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const resp =
+        type === 'signin' ? await signInUser(email, password) : await signUpUser(email, password);
+      setCurrentUser(resp);
+      //   console.log('Auth.js', resp);
+    } catch {
+      setErrorMessage('Sign in failed, Try again.');
+    }
+  };
+
+  return (
+    <div>
```

Comment:
Unnecessary div. Use fragments when trying to return many elements as one and avoid rendering new dom elements just to achieve that. Now, if this had a class attached to it and/or used for a style purpose (like in a flex) that would be okay.